exe/cli/cmd/deals_store.go

```diff
@@ -0,0 +1,106 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/caarlos0/spin"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/textileio/filecoin/deals"
+	"github.com/textileio/filecoin/lotus/types"
+)
+
+func init() {
+	storeCmd.Flags().StringP("file", "f", "", "Path to the file to store")
+	storeCmd.Flags().StringSliceP("miners", "m", []string{}, "miner ids of the deals to execute")
+	storeCmd.Flags().IntSliceP("prices", "p", []int{}, "prices of the deals to execute")
+	storeCmd.Flags().StringP("address", "a", "", "wallet address used to store the data")
+	storeCmd.Flags().Uint64P("duration", "d", 0, "duration to store the data for")
+
+	dealsCmd.AddCommand(storeCmd)
+}
+
+var storeCmd = &cobra.Command{
```

Comment:
So this command is similar to the fully UX one, but not interactive?

Reply:
Right. It is meant to be used stand alone. In theory, you'd use the other stand alone commands (`deals asks` and `deals watch`) before and after `deals store`.